src/analysis/retail/priest/shadow/modules/resources/InsanityUsage.tsx

```diff
@@ -2,10 +2,14 @@ import { t } from '@lingui/macro';
 import { formatPercentage } from 'common/format';
 import SPELLS from 'common/SPELLS';
 import RESOURCE_TYPES from 'game/RESOURCE_TYPES';
+import { Panel } from 'interface';
+import ResourceBreakdown from 'parser/shared/modules/resources/resourcetracker/ResourceBreakdown';
 import { SpellLink } from 'interface';
 import { ResourceLink } from 'interface';
 import Analyzer from 'parser/core/Analyzer';
 import { When, ThresholdStyle } from 'parser/core/ParseResults';
+//import ResourceBreakdown from 'parser/shared/modules/resources/resourcetracker/ResourceBreakdown';
```

Comment:
These can be removed as they were added above.

Reply:
I have removed both of those.